resources/views/projects/create.blade.php

```diff
@@ -12,76 +12,68 @@
-				<option value="">e.g. Web development class</option>
+				<option value="">e.g. Web Development</option>
 				@if(isset($courses))
 					@foreach($courses->all() as $course)
-						<option value="{{ $course->id }}">{{ $course->name }}</option>
+						<option value="{{ $course->id }}" {{ (old('course') == $course->id ? 'selected' : '') }}>{{ $course->name }}</option>
 					@endforeach
 				@endif
 			</select>
 		</div>
 		<!-- Associated team -->
-		<div class="field">
-			<label for="teamName">Associated Team</label>
-			<div class="fields">
-				<div class="seven wide field">
-					<select name="teamId" class="ui search dropdown {{ $errors->has('teamId') || $errors->has('bothTeams') ? 'error': '' }}" value="{{ old('teamId') }}">
-						<option value="">Associate an existing team</option>
-						@if(isset($teams))
-							@foreach($teams->all() as $team)
-								<option value="{{$team->id}}">{{$team->name}}</option>
-							@endforeach
-						@endif
-					</select>
-				</div>
-				<div class="two wide field">
-					<p style="font-weight: bold; font-size: 2rem; color: black; text-align: center;">OR</p>
-				</div>
-				<div class="seven wide field {{ $errors->has('bothTeams') ? 'error': '' }}">
-					<input type="text" name="createTeam" placeholder="Create a new team" value="{{ old('createTeam') }}">
+		<div class="fields">
+			<div class="seven wide field {{ $errors->has('existingTeam') || $errors->has('bothTeams') ? 'error': '' }}">
+				<label for="existingTeam">Choose an existing team</label>
+				<select id="existingTeam" name="existingTeam" class="ui search dropdown team" value="{{ old('existingTeam') }}" onchange="selectTeam()">
+					<option value="">My teams</option>
+					@if(isset($teams))
+						@foreach($teams->all() as $team)
+							<option value="{{ $team->id }}">{{ $team->name }}</option>
+						@endforeach
+					@endif
+				</select>
+			</div>
+			<div class="two wide two-column-divider">
+				<p style="font-weight: bold; font-size: 2rem; color: black; text-align: center; margin-top: 50%">OR</p>
+			</div>
+			<div class="seven wide field {{ $errors->has('newTeam') || $errors->has('bothTeams') ? 'error': '' }}">
+				<div class="field associatedTeam">
+					<label for="newTeam">Create a new team</label>
+					<input id="newTeam" type="text" name="newTeam" placeholder="e.g. New team" value="{{ old('newTeam')}}" onchange="createNewTeam()">
 				</div>
 			</div>
 		</div>
 		<!-- Labels -->
 		<div class="field {{ $errors->has('label') ? 'error': '' }}">
-			<label for="label">Labels</label>
+			<label for="labels">Labels</label>
 			<select name="labels[]" id="labels" multiple class="ui search dropdown">
 				<option value="">e.g. Finance</option>
 				@if(isset($labels))
 					@foreach($labels->all() as $label)
-						<option value="{{ $label->id }}">{{ $label->name }}</option>
+						<option value="{{ $label->id }}" {{ (collect(old('labels'))->contains($label->id)) ? 'selected' : '' }}>{{ $label->name }}</option>
```

Comment:
Este `old` yo lo puse en mi PR #110. I think it's slightly more efficient, pero t&uacute; decide. Igual el de `skillsets`.